src/Command/ArrayGeneratorCommand.php

```diff
@@ -53,7 +53,7 @@ protected function configure(): void
                 's',
                 InputOption::VALUE_OPTIONAL,
                 'Save generated array to generated-collections.json file.',
-                true
```

Comment:
Why you don't want to save file by default?

Reply:
because its clutter

Reply:
do you have any idea how to clean up this file or the file is not necessary?